source/PlayerInfo.cpp

```diff
@@ -3867,6 +3905,30 @@ void PlayerInfo::Save(const string &path) const
 	}
 	out.EndChild();
 
+//  Save a list of all known outfits to the player
+	WriteSorted(knownOutfits,
+		[](const Outfit *const *lhs, const Outfit *const *rhs)
+			{ return (*lhs)->TrueName() < (*rhs)->TrueName(); },
+		[&out](const Outfit *outfit)
+		{
+			out.Write(KNOWN_OUTFIT_KEY, outfit->TrueName());
+		});
+
+// Save a list of all visited outfitters
```

Comment:
gotta tab this the same as the rest, as with all comments

Reply:
will fix